src/backends/gr.jl

```diff
@@ -542,7 +542,7 @@ end
 
 function _cbar_unique(values, propname)
     out = last(values)
-    if any(x != out for x in values)
+    if any(map(x -> x != out, values))
```

Comment:
that is turning something that does not allocate into something that does
```julia
julia> f(val) = any(x != 1 for x in val)
f (generic function with 1 method)

julia> g(val) = any(map(x -> x != 1, val))
g (generic function with 1 method)

julia> @btime f($(collect(1:100)))
  1.936 ns (0 allocations: 0 bytes)
true

julia> @btime g($(collect(1:100)))
  28.060 ns (1 allocation: 160 bytes)
```

Reply:
I missed that it was a generator, will fix later.

Reply:
https://github.com/JuliaPlots/Plots.jl/pull/4470